mysql-test/columnstore/basic/t/mcol4202-crossdb-rename.test

```diff
@@ -0,0 +1,23 @@
+#
+# Test ALTER TABLE schemas in various possible ways.
+#
+-- source ../include/have_columnstore.inc
+-- source include/have_innodb.inc
+
+--disable_warnings
+DROP DATABASE IF EXISTS db1;
```

Comment:
I suggest you to rename db1 to mcol4202_db1 so there will be no overlap if MTR test fails and doesn't remove the leftovers.